tests/test_prompts.py

```diff
@@ -100,6 +100,59 @@ def test_generic_math_problem_augmentation_prompt():
     assert prompt.fill({'problem': "What's the meaning of life?"}) == expected_prompt
 
 
+def test_qwen_math_prompt_dict():
+    prompt = get_prompt('qwen/math-cot', 'qwen-instruct')
+
+    filled = prompt.fill({'problem': "What's the meaning of life?"}, return_templated_dict=True)
+    expected_result = [
+        {
+            'role': 'system',
+            'content': '<|im_start|>system\nPlease reason step by step, and put your final answer within \\boxed{}.<|im_end|>\n',
```

Comment:
are you sure that's how this is supposed to be in nemo-rl? They want the content to be formatted, but still be presented as messages list?